test/runtests.jl

```diff
@@ -10,5 +10,8 @@ testfiles = [ "dsp.jl", "util.jl", "windows.jl", "filter_conversion.jl",
 seed!(1776)
 
 for testfile in testfiles
-    eval(:(@testset $testfile begin include($testfile) end))
+    @testset "$testfile" begin
+        time = @elapsed eval(:(include($testfile)))
```

Comment:
When I did this I didn't consider whether compilation would be included in this @elapsed check. Do you know whether that is the case? If so, any idea how I can compile beforehand without running the tests, such that I only time the tests.